### DISCUSSION TITLE
Containers won't start after updating

### DISCUSSION THREAD

**Discussion by mio-19** [nextcloud/all-in-one#4044] [Questions] [answered]
### Steps to reproduce
1. Stop containers and Start and update containers
2. retry `Stop containers and Start and update containers` many more times
3. Stop and remove and recreate nextcloud-aio-mastercontainer
4. Manually `docker image pull` all nextcloud-aio containers 
5.  retry `Stop containers and Start and update containers` 

### Expected behavior 

Start

### Actual behavior 

Only two containers are running:
+ Collabora 
+ Talk

![image](https://github.com/nextcloud/all-in-one/assets/57285379/0f8f6d59-bb1c-4f2e-96ed-8a33af1a275c)
 
### Host OS 

Archlinux

#### Nextcloud AIO version 

Now v7.9.1

#### Current channel 

latest

**Comment by szaimen** [comment]
Hi, can you please post the mastercontainer logs here?

**Comment by mio-19** [comment, marked as answer]
Converted to manual-install
